src/components/Charts/ChartZones.tsx

```diff
@@ -94,7 +94,7 @@ const ChartZones = ({
 
   const lastPoint = last(data);
   const lastPointY = getY(lastPoint);
-  const lastPointZone = getZoneByValue(lastPointY, zones);
+  const lastPointRegion = getZoneByValue(lastPointY, zones);
```

Comment:
Hrm... I'm not 100% sure why you renamed this.  It seems like the actual type is `LevelInfo` which I think is the type we use for zones, but TBH I'm a little fuzzy on levels vs zones vs regions. :) Perhaps another thing to consider cleaning up at some point (not in this PR).

Reply:
I need to rename it back - I tried implementing `getRegionByValue` since regions already have the upper and lower limit, but then decided against it because regions don't have an explicit sense of order. For example, in `getRegionByValue`, I would have to do

```js
if (value <= region[0].valueFrom) {
  return region[0];
}
```

which implicitly assumes that `region[0]` corresponds to "Low". The code for zones is more explicit in that regard

```js
if (value <= zones[Level.LOW].upperLimit) {
  return zones[Level.LOW];
}
```

so I changed it back to use `zones`. I'm going to rename it right away, it's a pretty small change.